src/scripts/main.js

```diff
@@ -18,10 +18,30 @@ const food = {
   },
 };
 
-const tree = document.querySelector('#tree');
+let tree = document.querySelector('#tree');
+
+if (!tree) {
+  tree = document.createElement('ul');
+  tree.id = 'tree';
+  document.body.appendChild(tree);
+}
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+    ul.appendChild(li);
+  }
+  element.appendChild(ul);
+
+  return element;
 }
```

Comment:
The 'createTree' function is supposed to be a recursive function that generates nested lists for nested objects. However, your current implementation has a problem: when you call 'createTree' recursively in line 38, you're not doing anything with the result. The function generates a nested list, but then this list is not attached to anything and gets lost. This is a bug and needs to be fixed. You should append the result of the recursive call to the current list item. This can be done using the appendChild method.